diff --git a/buildscripts/vcpkg/triplets/x64-linux.cmake b/buildscripts/vcpkg/triplets/x64-linux.cmake
index 882213456..caedd9d11 100644
--- a/buildscripts/vcpkg/triplets/x64-linux.cmake
+++ b/buildscripts/vcpkg/triplets/x64-linux.cmake
@@ -1,6 +1,7 @@
 set(VCPKG_TARGET_ARCHITECTURE x64)
-set(VCPKG_CRT_LINKAGE dynamic)
+set(VCPKG_CRT_LINKAGE static)
 set(VCPKG_LIBRARY_LINKAGE static)
 
 set(VCPKG_CMAKE_SYSTEM_NAME Linux)
+set(VCPKG_BUILD_TYPE release)
 
